test/verify/check-dashboard

```diff
@@ -204,7 +204,7 @@ class TestBasicDashboard(MachineCase, DashBoardHelpers):
         b.leave_page()
         b.wait_js_cond('ph_select("#machine-dropdown ul li").length == 3')
 
-        if m.image != "rhel-7-4":
+        if m.image != "rhel-7-5":
```

Comment:
I don't understand this change.  What has changed about rhel-7.5 so that it is now exceptional, while it wasn't before?

Reply:
Up to now we have tested RPMs from master on the rhel-7-5 image. Now rhel-7-5 takes the role of the previous rhel-7-4 where we test against the preinstalled cockpit-* packages from RHEL base. That is version 154, thus before the mobile navigation changes and a few other features. Thus we now need to have these "old version" special cases for 7.5

Reply:
Thanks, got it.